src/BundleAnalyzerPlugin.js

```diff
@@ -55,14 +55,21 @@ class BundleAnalyzerPlugin {
 
       if (actions.length) {
         // Making analyzer logs to be after all webpack logs in the console
-        setImmediate(() => {
-          actions.forEach(action => action());
+        setImmediate(async () => {
+          try {
+            for (let i = 0; i < actions.length; ++i) await actions[i]();
+            if (typeof callback === 'function') callback();
```

Comment:
Let's replace these `if` conditions with `callback = callback || () => {};` at the top of the `done` function.
1/3